### DISCUSSION TITLE
What account for a VPC Inspection

### DISCUSSION THREAD

**Discussion by Martinagsr** [aws-samples/aws-secure-environment-accelerator#993] [Q&A] [answered]
What is your recommandation for the account of the VPC Inspection?

**Comment by Brian969** [comment]
Lots of variables, but, primary question is What traffic are you inspecting? north-south, east-west, both, other?

**Reply by Martinagsr** [reply]
Traffic from client site (Direct Connect) at the first step and maybe the east-west traffic. The firewall appliance will acces Internet often for updating security services like IPS and antivirus.

**Comment by Brian969** [comment, marked as answer]
Sorry, I should have first pointed you to [FAQ's 1.7.x](https://aws-samples.github.io/aws-secure-environment-accelerator/faq/#17-network-architecture) (and I have not reviewed recently, so it likely still does not answer your full question, but is a valuable starting point - I'll also re-review the FAQ).
